src/services/db/GitFileCommitService.ts

```diff
@@ -258,6 +258,49 @@ export default class GitFileCommitService {
     this.pushToGithub(sessionData, shouldUpdateStagingLite)
   }
 
+  async deleteMultipleFiles(
+    sessionData: UserWithSiteSessionData,
+    githubSessionData: GithubSessionData,
+    { items }: { items: Array<{ filePath: string; sha: string }> }
+  ): Promise<void> {
+    logger.info(
+      `Deleting multiple files in local Git file system for repo: ${sessionData.siteName}`
+    )
+
+    const stagingDeleteResult = await this.gitFileSystemService.deleteMultipleFiles(
+      sessionData.siteName,
+      items,
+      sessionData.isomerUserId,
+      STAGING_BRANCH
+    )
+
+    const shouldUpdateStagingLite =
```

Comment:
staging lite doesn't contain images or files right? in that case why do we need to update stg lite branch?

Reply:
We don't have that guarantee at this stage, because we only know the `filePath` and `sha`. Doing this is safer in case we somehow reuse this function to bulk delete pages in the future.